code/modules/mob/living/carbon/carbon.dm

```diff
@@ -428,7 +428,7 @@
 		return 0
 	return ..()
 
-/mob/living/carbon/proc/vomit(lost_nutrition = 10, blood = FALSE, stun = TRUE, distance = 1, message = TRUE, toxic = FALSE, harm = TRUE, force = FALSE)
+/mob/living/carbon/proc/vomit(lost_nutrition = 10, blood = FALSE, specialcolor = null, stun = TRUE, distance = 1, message = TRUE, toxic = FALSE, harm = TRUE, force = FALSE)

```

Comment:
you cant just jam a new param in the middle of a proc, this'll break most of the calls to this proc because the third param is now special_color instea of stun, which means almost every instance of vomit() will be broken.

Reply:
fixed